### ISSUE TITLE
Support for Cypress 10

### ISSUE THREAD

**Issue by mmitchell3590** [Shelex/cypress-allure-plugin#143]
**Is your feature request related to a problem? Please describe.**
We recently updated to Cypress version 10, and it seems like the allure-results are no longer generating. Are there some changes needed in the plugin configuration?

**Describe the solution you'd like**
Support for Cypress 10 and greater

**Comment by tulavma** [comment]
I have the same problem. I updated to version 10 of Cypress and Allure stopped generating the results (folder allure-results). I think this is a very high priority improvement given the wide use of Allure with Cypress. Without the adjustment, you will not be able to continue using Allure.

**Comment by DeChrish** [comment]
@mmitchell3590 @tulavma  - Pl use the new version and test it

**Comment by Shelex** [comment]
> @mmitchell3590 @tulavma - Pl use the new version and test it

Why?

**Comment by roydekleijn** [comment]
@Shelex can we help? or what needs to be done in order to make this compatible with Cypress 10?

**Comment by IrfanSyed-PSC** [comment]
Same problem. We see that working fine on macos but not on windows.

**Comment by AlexandreQue** [comment]
Hello,

try this implementation in your `cypress.config.js/ts` : 
```javascript
e2e: {
    // We've imported your old cypress plugins here.
    // You may want to clean this up later by importing these.
    setupNodeEvents(on, config) {
      allureWriter(on, config);
      return require('./cypress/plugins/index.ts')(on, config);
    },
```

**Comment by IrfanSyed-PSC** [comment]
Thank you, that worked. I should have tried it myself before, but did not think about it as it was working in macOS.

**Comment by AlexandreQue** [comment]
@IrfanSyed-PSC Your Welcome
@Shelex I think you can close this thread

Have a nice day.
Best regards.

**Comment by Shelex** [comment]
@AlexQuei thanks for resolving :D
Actually I have checked cypress 10 and changelog, found potential issue so prepared a fix (v2.28.0) as well as updated docs to match latest version.
